tests/integration/network/auction_test.go

```diff
@@ -0,0 +1,240 @@
+		context.Background(),
+		bz,
+	)
+	require.NoError(s.T(), err)
+	require.Equal(s.T(), uint32(0), res.CheckTx.Code)
+	require.Equal(s.T(), uint32(0), res.TxResult.Code)
+
+	s.Run("two valid bids--balance/fee verification", func() {
+		// Store the receiver's initial balance
+		beginReceiverBalances, err := s.NetworkSuite.Balances(*receiver)
+		require.NoError(s.T(), err)
+		beginReceiverBalance := beginReceiverBalances.AmountOf(params.Params.ReserveFee.Denom)
+
+		bid1Seq, _, err := getAccount(context.Background(), authtypes.NewQueryClient(cc), *bidder1)
+		s.Require().NoError(err)
+		bid2Seq, _, err := getAccount(context.Background(), authtypes.NewQueryClient(cc), *bidder2)
+		s.Require().NoError(err)
+
+		// Get current height
+		resp, err := cmtClient.Status(context.Background())
+		s.Require().NoError(err)
+		bidHeight := uint64(resp.SyncInfo.LatestBlockHeight + 1)
+
+		// Bidder1's send tx they want included
+		send1Tx, err := s.NetworkSuite.CreateTxBytes(
+			context.Background(),
+			network.TxGenInfo{
+				Account:          *bidder1,
+				GasLimit:         1000000,
+				TimeoutHeight:    bidHeight,
+				Fee:              fee,
+				Sequence:         bid1Seq + 1,
+				OverrideSequence: true,
+			},
+			banktypes.NewMsgSend(bidder1.Address(), receiver.Address(), sdk.NewCoins(sdk.NewCoin(params.Params.ReserveFee.Denom, math.NewInt(1)))),
+		)
+		require.NoError(s.T(), err)
+
+		// Bidder1's Bid Tx
+		bid1Tx, err := s.NetworkSuite.CreateTxBytes(
+			context.Background(),
+			network.TxGenInfo{
+				Account:          *bidder1,
+				GasLimit:         1000009,
+				TimeoutHeight:    bidHeight,
+				Fee:              fee,
+				Sequence:         bid1Seq,
+				OverrideSequence: true,
+			},
+			auctiontypes.NewMsgAuctionBid(
+				bidder1.Address(),
+				params.Params.ReserveFee,
+				[][]byte{send1Tx},
+			),
+		)
+		require.NoError(s.T(), err)
+
+		// Bidder2's send tx they want included
+		send2Tx, err := s.NetworkSuite.CreateTxBytes(
+			context.Background(),
+			network.TxGenInfo{
+				Account:          *bidder2,
+				GasLimit:         1000000,
+				TimeoutHeight:    bidHeight,
+				Fee:              fee,
+				Sequence:         bid2Seq + 1,
+				OverrideSequence: true,
+			},
+			banktypes.NewMsgSend(bidder2.Address(), receiver.Address(), sdk.NewCoins(sdk.NewCoin(params.Params.ReserveFee.Denom, math.NewInt(2)))),
+		)
+		require.NoError(s.T(), err)
+
+		// Bidder2's Bid Tx
+		bid2Tx, err := s.NetworkSuite.CreateTxBytes(
+			context.Background(),
+			network.TxGenInfo{
+				Account:          *bidder2,
+				GasLimit:         1000000,
+				TimeoutHeight:    bidHeight,
+				Fee:              fee,
+				Sequence:         bid2Seq,
+				OverrideSequence: true,
+			},
+			auctiontypes.NewMsgAuctionBid(
+				bidder2.Address(),
+				params.Params.ReserveFee.Add(params.Params.MinBidIncrement),
+				[][]byte{send2Tx},
+			),
+		)
+		require.NoError(s.T(), err)
+
+		// Broadcast the bids
+		for _, tx := range [][]byte{bid1Tx, bid2Tx} {
```

Comment:
q why can't we synchronously broadcast both bids (BroadcastTxCommit)?

Reply:
nvm, we're expecting that bid2 is the only bid committed?

Reply:
Yeah only one should actually commit.